support/globalconfig/ingress.go

```diff
@@ -18,12 +17,20 @@ func IngressConfig() *configv1.Ingress {
 }
 
 func ReconcileIngressConfig(cfg *configv1.Ingress, hcp *hyperv1.HostedControlPlane, globalConfig GlobalConfig) {
-	cfg.Spec.Domain = IngressDomain(hcp)
+	cfg.Spec.Domain = IngressDomain(hcp, globalConfig.Ingress)
 	if globalConfig.Ingress != nil {
 		cfg.Spec = globalConfig.Ingress.Spec
 	}
 }
 
-func IngressDomain(hcp *hyperv1.HostedControlPlane) string {
-	return fmt.Sprintf("apps.%s", BaseDomain(hcp))
+func IngressDomain(hcp *hyperv1.HostedControlPlane, ingressConfig *configv1.Ingress) string {
+	if ingressConfig != nil {
+		if len(ingressConfig.Spec.AppsDomain) > 0 {
+			return ingressConfig.Spec.AppsDomain
+		}
+		return ingressConfig.Spec.Domain
+	} else {
```

Comment:
really minor nit: we don't need an else here

Reply:
totally makes sense fixed